### ISSUE TITLE
SA_SHOW_BROWSER set to true causes errors.

### ISSUE THREAD

**Issue by magicaltoast** [ulixee/secret-agent#266]
Turning SA_SHOW_BROWSER causes the program to crash when it isn't set program works as expected

Node version: 14.17.1 
Secret agent version: 1.5.1
OS: Ubuntu 20.04

**Code**
```js
const agent = require('secret-agent');

process.env.SA_SHOW_BROWSER = 'true';

(async () => {

    // configure input.url by running as node default-agent.js --input.url="https://ulixee.org"
    const url = agent.input.url ?? `https://dataliberationfoundation.org/`;

    const start = new Date();

    await agent.goto(url, 5e3);
    await agent.waitForPaintingStable();

    agent.output = {
        url,
        start,
        end: new Date(),
    };

    await agent.close();
})();
```
**Error**
```
 UnhandledErrorOrRejection {
  clientError: 'TypeError: coreTab.goto is not a function',
  context: {},
  sessionId: null,
  sessionName: undefined
} TypeError: coreTab.goto is not a function
    at Tab.goto (/home/idk/WebstormProjects/secret_agent_test/node_modules/client/lib/Tab.ts:161:36)
    at /home/idk/WebstormProjects/secret_agent_test/index.js:12:5
```

**Comment by blakebyrnes** [comment]
Thanks for the report. I'll get the fix into the next build.

**Comment by blakebyrnes** [comment]
fixed in version 1.5.2
